File: c++/Arrays/max_min_array.cpp
#include<iostream>
using namespace std;

int max_array(int arr[],int n)
{
	int max = arr[0];
	for(int i=0;i<n;i++)
	{
		if(arr[i] > max)
			max = arr[i];
	}
	return max;
}

int min_array(int arr[],int n)
{
	int min = arr[0];
	for(int i=0;i<n;i++)
	{
		if(arr[i]<min)
			min=arr[i];
	}
	return min;
}


int main()
{
	int n;
	cout<<"Enter the size of the array = ";
	cin >> n;
	int arr[n];
	cout<<"Enter the elements of the array = ";
	for(int i=0;i<n;i++)
		cin>>arr[i];
	int max = max_array(arr,n);
	int min = min_array(arr,n);
	cout<<"\nMaximum element is  =  "<<max;
	cout<<"\nMinimum element is  =  "<<min;
}

File: Python/Arrays/max_min_array.py
def minimum_element(A,size):
    min = A[0]
    for i in range(0,size):
        if(min > A[i]):
            min = A[i]
    return min


def maximum_element(A,size):
    max = A[0]
    for i in range(0,size):
        if(max < A[i]):
            max = A[i]
    return max
    

A = []
n = int(input("Size of array = "))
for i in range(0,n):
    ele = int(input())
    A.append(ele)
min = minimum_element(A,n)
print("Minimum element in the array = ",min)
max = maximum_element(A,n)
print("Maximum element in the array = ",max)
